src/util.cpp

```diff
@@ -427,6 +427,19 @@ std::vector<std::string> ArgsManager::GetArgs(const std::string& strArg)
     return {};
 }
 
+std::vector<std::string> ArgsManager::GetUniqueArgs(const std::string& strArg)
+{
+    LOCK(cs_args);
+    std::vector<std::string> args;
```

Comment:
I was tempted to name it `GetStableUniqueArgs` but maybe it's too much for most of you.